### PR TITLE
Story/cord 25

### PR DESCRIPTION
**Pull request by aa-kashk**
Show all available metadata on paper page.

Most of the metadata is already being displayed on the paper html page

### CODE REVIEW

File: cord19-explorer/src/main/webapp/WEB-INF/views/paper.html [hunk 1]
```diff
@@ -33,6 +33,8 @@ <h3>[[${#strings.defaultString(publication.metadata.title, 'No title available')
    		 <span th:if="${author.affiliation.locationRegion != null}" th:remove="tag">[[${author.affiliation.locationRegion}]],</span> 
    		 <span th:if="${author.affiliation.locationCountry != null}" th:remove="tag">[[${author.affiliation.locationCountry}]]</span>
    		 <span th:if="${author.affiliation.selectedWikiarticle != null}"><span class="label label-info">matched</span> <i>[[${author.affiliation.selectedWikiarticle.title}]]</i> ([[${author.affiliation.selectedWikiarticle.coordinates}]])</span>
+   		 <span th:if="${author.email != null}">[[${author.email}]] </span>
+   		 <span th:if="${author.uri != null}">[[${author.uri}]] </span>
```
Comment (aa-kashk) [inline cord19-explorer/src/main/webapp/WEB-INF/views/paper.html:37]: This page already includes most fields from MetaDataImpl, PersonImpl and AffiliationImpl types.

**Review by prjadhav14 — APPROVED** [review]

File: cord19-explorer/src/main/webapp/WEB-INF/views/paper.html (hunk 1 above)
Comment (jdamerow) [reply cord19-explorer/src/main/webapp/WEB-INF/views/paper.html:37]: there is a lot more metadata on the publication object (e.g. journal, doi, pmcid, etc)

**Comment by jdamerow** [comment]
Jenkins, test this please.

**Comment by jdamerow** [comment]
Jenkins, test this please.

**Comment by jdamerow** [comment]
Jenkins, test this please.

**Comment by jdamerow** [comment]
let's put the extra metadata in a (stripped) table;

**Comment by jdamerow** [comment]
Jenkins, test this please.

**Comment by jdamerow** [comment]
property name should be one column, property value should be a second. it's really hard to see right now which values are set.

**Comment by jdamerow** [comment]
Jenkins, test this please.

**Pull request merged**